packages/components/cascader-panel/src/index.vue

```diff
@@ -26,9 +26,9 @@ import {
   watch,
 } from 'vue'
 import isEqual from 'lodash/isEqual'
+import { isClient } from '@vueuse/core'
```

Comment:
**Question:** Why do we need to use this in a 3rd party library? Are we trying to couple this project with `@vueuse` or what?

Reply:
We have used `isClient` in vueuse. We can remove it.